Automattic/wp-calypso
# Domains: Extend primary domain restriction to mappings

#### Changes proposed in this Pull Request

This PR extends the primary domain on a free site restrictions to mapped external domains as well.

Depends on D41610-code

#### Testing instructions

* Plan upsell should appear when transfer is added to cart on a free site
* When downgrading to free plan, you should see a warning if the primary domain is mapped
* You should see the prompt to upgrade when trying to set your mapping as primary
